WordPress/src/main/java/org/wordpress/android/models/Blog.java

```diff
@@ -515,7 +517,30 @@ public void setPlanID(long planID) {
     public String getPlanShortName() {
         return StringUtils.notNullStr(planShortName);
     }
+
     public void setPlanShortName(String name) {
         this.planShortName = StringUtils.notNullStr(name);
     }
+
+    public String getCapabilities() {
```

Comment:
> I honestly didn't know about this rule, I'll try to follow it from now on.

I should mention that this isn't a rule for the entire app - it's just something I've found useful, to avoid having to check the result for `null` all the time.